coderd/templates_test.go

```diff
@@ -1326,3 +1327,57 @@ func TestTemplateMetrics(t *testing.T) {
 		dbtime.Now(), res.Workspaces[0].LastUsedAt, time.Minute,
 	)
 }
+
+func TestNotifyDeletedTemplate(t *testing.T) {
+	t.Parallel()
+
+	t.Run("OnlyNotifyOwnersAndTemplateAdmins", func(t *testing.T) {
```

Comment:
Please add a separate test to validate the functionality in isolation; you can then reduce some complexity in this test and keep it nicely scoped.

Reply:
Not sure if I got it. What should I separate and what complexity does it solve?